src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$backroundColor: #eee;
+$cellSize: 100px;
+$cellGap: 1px;
+$calendarPadding: 10px;
+$cellFonts: "Arial", "Helvetica", sans-serif;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cellSize + $calendarPadding * 2;
+  padding: 10px;
+  gap: 1px;
+
+  $days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
+  @each $day, $number in $days {
+    &--start-#{$day} &__cell:first-child {
+      margin-left: $cellSize * $number + $cellGap * $number;
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-#{$i} &__cell:nth-child(n + #{$i}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 30px;
+    font-family: $cellFonts;
+    box-sizing: border-box;
+    background-color: $backroundColor;
+    width: $cellSize;
+    height: $cellSize;
+    border: 1px solid #000;
+    transition-duration: 0.5s;
+  }
+
+  &__cell:hover {
+    transform: translate(0, -20px);
```

Comment:
0 is not necessary in `translate` in this case

Reply:
if you do not set 0, then the cell moves to the left, not up

Reply:
Cause you need to set Y axis to this property Should use like so `transform: translateY(-20px);`